client/src/components/Common/CertificationItem/styles.css

```diff
@@ -0,0 +1,23 @@
+.certification-container .image {
+  width: 100%;
+  border-radius: 5px;
+  box-shadow: 0px 0px 15px rgba(0,0,0,0.1);
+  cursor: pointer;
+}
+
```

Comment:
Most of the CSS files do not have spaces between the statements... Myabe to be consistent it is better to remove them everywhere... 🤔